sympy/physics/vector/dyadic.py

```diff
@@ -55,6 +56,12 @@ def __init__(self, inlist):
                 i -= 1
             i += 1
 
+    @property
+    def func(self):
```

Comment:
This being a public property needs a docstring.

Reply:
okay I'll add that.